src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -68,7 +83,55 @@ public FilterPanel(UI ui, IModel model, PanelControl parentPanelControl, int pan
                 getStyleClass().remove("panel-focused");
             }
         });
-        
+        setupKeyboardShortcuts();
+    }
+
+    private void setupKeyboardShortcuts() {
+        addEventHandler(KeyEvent.KEY_PRESSED, event -> {
+            if (MAXIMIZE_WINDOW.match(event)) {
+                ui.maximizeWindow();
+            }
+            if (MINIMIZE_WINDOW.match(event)) {
+                ui.minimizeWindow();
+            }
+            if (DEFAULT_SIZE_WINDOW.match(event)) {
+                ui.setDefaultWidth();
+            }
+            if (SWITCH_DEFAULT_REPO.match(event)) {
```

Comment:
This can be removed as I made it a global keyboard shortcut (i.e. binded to Scene) in #1009.